doc/uftrace.html

```diff
@@ -274,6 +275,10 @@
 ...     perf_event: [ .green[on]  ] - perf (PMU) event support
 ...       schedule: [ .green[on]  ] - scheduler event support
 ...       capstone: [ .green[on]  ] - full dynamic tracing support
+...      libunwind: [ .red[OFF] ] - stacktrace support support (optional for debugging)
+
+# If you want to turn on libunwind option
+$ sudo apt-get install -y libunwind-dev
```

Comment:
This tutorial slide doesn't have to cover all the details so no need to have this change.